docs/platforms/javascript/common/tracing/distributed-tracing/dealing-with-cors-issues/index.mdx

```diff
@@ -16,8 +16,94 @@ notSupported:
   - javascript.nestjs
 ---
 
-If your frontend and backend are hosted on different domains (for example, your frontend is on `https://example.com` and your backend is on `https://api.example.com`), and the frontend does XHR/fetch requests to your backend, you'll need to configure your backend [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers) headers to ensure requests aren't blocked.
+If your frontend and backend are hosted on different domains (for example, your frontend is on `https://example.com` and your backend is on `https://api.example.com`), you may need to configure your backend [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers) headers to ensure distributed tracing works properly.
 
-Configure your backend CORS to allow the `sentry-trace` and `baggage` headers.
+## Understanding Trace Propagation
 
-Your server's response header configuration might look like: `"Access-Control-Allow-Headers: sentry-trace, baggage"`. Your configuration will be specific to your setup.
+Sentry doesn't attach the `sentry-trace` and `baggage` headers to every outgoing request. Instead, it only attaches these headers to requests whose URLs match the patterns specified in the `tracePropagationTargets` configuration option.
+
+**By default**, `tracePropagationTargets` is set to `['localhost', /^\//]`, which means trace headers are only attached to:
+- Requests containing `localhost` in their URL (e.g., `http://localhost:3000/api`)
+- Requests whose URL starts with `/` (e.g., `/api/users`, `/graphql`)
+
+This default behavior helps prevent sending trace data to third-party services and reduces potential CORS issues.
```

Comment:
This should not reduce issues but avoid them. Nothing can happen with the default config, basically.